app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.component.ts

```diff
@@ -65,17 +65,22 @@ export class TravisTriggerOptions {
   };
 }
 
+class TravisTriggerOptionsComponent implements IComponentOptions {
+  public bindings: any;
+  public templateUrl: string;
+  public controller: any;
+
+  constructor() {
+    this.bindings = {
+      command: '='
+    };
+    this.templateUrl = require('./travisTriggerOptions.component.html');
+    this.controller = TravisTriggerOptions;
+  }
+}
```

Comment:
`templateUrl` expects a URL - hence the `require` - as @anotherchrisberry noted, you want `template` and then you can use inline.  But as @danielpeach also pointed out, you have a recursive dependency.